broadinstitute/single_cell_portal_core
# * SCP-2989 Handle no filename being passed in download URL

This is a PR to address [sentry error](https://sentry.io/organizations/broad-institute/issues/2031252931/?project=1424198&statsPeriod=14d). The error was caused by a user trying to download a file via URL where the did have provided a filename. 

To test:

- Go to "Download" tab on a public study.
- Copy link address on any study download option
- Post link in browser
- Remove "filename= </file name>" from URL. Ex: https://localhost/single_cell/data/public/SCP1/<public study name>. There's a clearer example of this in the URL in the screenshot.
- Hit enter

Screenshot of the error a user will see when the filename is not included in the URL
<img width="1672" alt="Screen Shot 2021-01-19 at 1 18 57 PM" src="https://user-images.githubusercontent.com/37722472/105076341-ede8a200-5a58-11eb-862c-aef5f6ffddc8.png">
